govc/datastore/rm.go

```diff
@@ -106,11 +107,8 @@ func (cmd *rm) Run(ctx context.Context, f *flag.FlagSet) error {
 		err = remove(ctx, args[0])
 	}
 
-	if err != nil {
-		if types.IsFileNotFound(err) && cmd.force {
-			// Ignore error
-			return nil
-		}
+	if errors.Is(err, &types.FileNotFound{}) && cmd.force {
```

Comment:
We have to use different pointer/non-pointer semantics due to the way the underlying errors are constructed